code/controllers/subsystem/processing/instruments.dm

```diff
@@ -0,0 +1,86 @@
+PROCESSING_SUBSYSTEM_DEF(instruments)
+	name = "Instruments"
+	init_order = INIT_ORDER_INSTRUMENTS
+	wait = 0.5
```

Comment:
I am a little concerned with having a subsystem fire this quickly

Reply:
There won't be that many instruments playing at once (maybe 5 tops) and the note timing is dependent on `world.tick_lag` (which is `0.5` by default I think?). Setting the subsystem delay to be higher than `world.tick_lag` would result in notes playing later than expected